openshift/e2e-common.sh

```diff
@@ -90,10 +90,16 @@ EOF
   export GOPATH=/tmp/go
   local failed=0
   pushd $operator_dir || return $?
-  OPENSHIFT_CI="true" make generated-files install-kafka || failed=$?
+  OPENSHIFT_CI="true" make generated-files install-operator || failed=$?
   popd || return $?
 
-  oc apply -f openshift/knative-eventing.yaml
+  local eventing_cr="$(mktemp -t eventing-XXXXX.yaml)"
+  cp openshift/knative-eventing.yaml "$eventing_cr"
+  enable_tracing "$eventing_cr"
+  oc apply -f "$eventing_cr"
+
+  # Install KnativeKafka after installing KnativeEventing with tracing enabled
+  oc apply -f openshift/knative-kafka.yaml
```

Comment:
Can we use the operator to deploy everything?

Reply:
with something like:
```
export TRACING_BACKEND=zipkin
OPENSHIFT_CI="true" make generated-files install-tracing install-kafka || failed=$?
```
I would expect it to work.

Reply:
The problem is that the knative-eventing custom resource from serverless-operator is different from knative-eventing in eventing-kafka-broker. The difference is this: https://github.com/openshift-knative/eventing-kafka-broker/blob/release-v1.4/openshift/knative-eventing.yaml#L7-L15 However, maybe I could add these lines via patch after knative-eventing and knative-kafka are installed? Will this cause any issues?

Reply:
OK. Let's try it, pushed a new commit.

Reply:
It doesn't work. The config-tracing configmap is empty. Will look into it tomorrow.

Reply:
So. Pushed another commit. Now I'm re-using the tracing and KnativeKafka installation from serverless-operator. And I'm installing a specific KnativeEventing from eventing-kafka-broker repo.